plugin/webhook/discord.go

```diff
@@ -54,7 +54,7 @@ func (receiver *DiscordReceiver) post(context Context) error {
 	embedList := []DiscordWebhookEmbed{}
 
 	fieldList := []DiscordWebhookEmbedField{}
-	for _, meta := range context.MetaList {
+	for _, meta := range context.genMeta() {
```

Comment:
genMeta -> getMetaList()

get() is a canonical and better name. MetaList is to be consistent with other List structured names.